editor/src/clj/editor/properties.clj

```diff
@@ -406,36 +408,46 @@
+                                (when-not ret
+                                  (tap> (map property-edit-type (val e))))
+                                ret)))
+                          ; Coalesce into properties consumable by e.g. the properties view
+                          (map (fn [[k v]]
+                                 (let [prop {:key k
+                                             :node-ids (mapv :node-id v)
+                                             :prop-kws (mapv (fn [{:keys [prop-kw]}]
+                                                               (cond (some? prop-kw) prop-kw
+                                                                     (vector? k) (first k)
+                                                                     :else k)) v)
+                                             :tooltip (some :tooltip v)
+                                             :values (mapv (fn [{:keys [value]}]
                                                              (when-not (g/error? value)
                                                                value)) v)
-                                              :errors  (mapv (fn [{:keys [value error]}]
-                                                               (or error
-                                                                   (when (g/error? value) value))) v)
-                                              :edit-type (property-edit-type (first v))
-                                              :label (:label (first v))
-                                              :read-only? (reduce (fn [res read-only] (or res read-only)) false (map #(get % :read-only? false) v))}
-                                        default-vals (mapv :original-value (filter #(contains? % :original-value) v))
-                                        prop (if (empty? default-vals) prop (assoc prop :original-values default-vals))]
-                                    [k prop]))
-                                common-props))]
+                                             :errors (mapv (fn [{:keys [value error]}]
+                                                             (or error
+                                                                 (when (g/error? value) value))) v)
+                                             :edit-type (property-edit-type (first v))
+                                             :label (:label (first v))
+                                             :read-only? (reduce (fn [res read-only] (or res read-only)) false (map #(get % :read-only? false) v))}
+                                       default-vals (mapv :original-value (filter #(contains? % :original-value) v))
+                                       prop (if (empty? default-vals) prop (assoc prop :original-values default-vals))]
+                                   (pair k prop)))))
+                        visible-prop-colls)]
```

Comment:
No change in logic, but I refactored this block to use transducers instead of lazy seqs when investigating the issue. The actual issue was that this block filters out properties with non-equal edit types, but we used anonymous functions for `:set-fn`/`:clear-fn` values in edit type.